ql/src/java/org/apache/hadoop/hive/ql/security/authorization/PrivilegeSynchronizer.java

```diff
@@ -78,6 +83,9 @@ private void addACLsToBag(
     for (Map.Entry<String, Map<HiveResourceACLs.Privilege, HiveResourceACLs.AccessResult>> principalAcls
         : principalAclsMap.entrySet()) {
       String principal = principalAcls.getKey();
+      int[] columnPrivilegeBits = new int[] {0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
can this be changed to array of booleans or char ?

Reply:
Yes, i will change this to booleans.